src/site/content/en/blog/coop-coep/index.md

```diff
@@ -23,27 +23,34 @@ origin_trial:
 feedback:
   - api
 ---
+{% Aside 'gotchas' %}
+**[15th Oct. 2020]**: `self.crossOriginIsolated` and Shared Array Buffer on Android
+Chrome are now available in Chrome 87. `document.domain` is now immutable when
```

Comment:
SABs will be launched in M88 not M87 on Android, no changes for Desktop yet.

Reply:
This should have been sorted. Please check.